pkg/tsdbctl/query.go

```diff
@@ -101,6 +101,10 @@ func (qc *queryCommandeer) query() error {
 		return err
 	}
 
+	if oneHourMillis%step != 0 && step%oneHourMillis != 0 {
+		return errors.New("interval step should be a divisor or a dividend of 1 hour. Example: 10m, 30m, 2h, etc..")
+	}
```

Comment:
We discussed removing this bit in today's status meeting, since we have to support such queries, and @yaronha has a fix for that.